recipes/czmq/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "czmq" %}
+{% set version = "4.0.2" %}
+{% set sha256 = "794f80af7392ec8d361ad69646fc20aaa284d23fef92951334009771a732c810" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/zeromq/czmq/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  skip: True  # [win]
```

Comment:
It looks like this package can be built on windows? Any reason to not attempt it here?

https://github.com/zeromq/czmq#building-on-windows

Reply:
Would probably go with CMake actually as that is easier to control which Visual Studio version is used.